chapter_12/3_Reconstruction.tex

```diff
@@ -0,0 +1,66 @@
+2. From a practical and algorithmic perspective, topological features of M are inferred from
+the nerve of the union of balls, using the Nerve Theorem.
+
+
+\subsection{Offsets and Sublevels} 
+
+\begin{definition}[Sublevel Set]
+Given a function $f$ with domain $X$, the sublevel set for $f$ at a constant r is given by $\{(x_1, x_2, ... x_n) \: | \: f(x_1, x_2, ... x_n) \leq r\}$
+\end{definition}
+
+\begin{definition}[r-offset]
+Given a compact subset K of $\bbR^d$ and a non-negative
+real number r, the r-offset of $K$, called $K^r$, is the the r-sublevel set of the distance function $d_K:\bbR^d \rightarrow \bbR$, such that $d_K(x) = \text{inf}_{y\in K}||x-y||$. That is, $K^r = \cup_{x\in{K}}B(x, r) = d_{k}^{-1}([0, r])$, the preimage of the radius interval.
+\end{definition}
+
+
+Change:
+This remark allows us to compare the topology of the offsets of compact sets that are close to each other. For example, the offsets of Xn and the support of the measure $\mu$, M, by their compactness and with a few mild assumptions on their Hausdorff difference, can be related up to homotopy equivalence. Specifically When M is a smooth compact submanifold there exists an r for which the offsets of Xn are homotopy equivalent to M.
+
+
+The r-offset 
+
+\begin{definition}
+A function $\phi:\bbR^d \rightarrow \bbR_+$ is called distance like if $x\rightarrow ||x||^2 -\phi^2(x)$ is convex and if the pre-image of any compact set in $\bbR$ is a compact set in $\bbR^d$. 
+\end{definition}
+
+
+Mention Isotopy Lemma Grove (1993)
+
+\begin{theorem}[Reconstruction Theorem]
+Let $\phi, \psi$ be two distance-like functions. Given $||\phi - \psi||_\infty < \epsilon$ and that $\text{reach}_\alpha(\phi) \geq R$ for some positive $\epsilon$ and $\alpha$, then $\forall{r} \in [4\epsilon/\alpha^2, R-3\epsilon]$ and $\forall{\eta}\in(0, R)$, the sublevel sets $\psi^r$ and $\phi^\eta$ are homotopy equivalent if $$\epsilon \leq \frac{R}{5+4/\alpha^2}$$
+\end{theorem}
+
+The Reconstruction theorem and the Nerve Theorem allow us to infer the topology of data from a simplicial complex built on top of an approximating finite sample. However, the Reconstruction Theorem requires a regularity assumption (through the $\alpha$-reach condition and the choice of a radius r for the ball used to build Cech$_r(\mathbb{X})$). Additionally, Cech$_r(\mathbb{X})$
+provides a simplicial complex that, while provably related to the underlying topological space, is often not computationally useful. As a solution, we introduce the idea of homology. Homology is a more general approach which associates algebraic objects to the properties of the topological space, allowing for more computable statistics over our constructed simplicial complexes. 
+
+
+Change:
+$\text{reach}_\alpha(\phi) \geq R$ can be interpreted as regularity condition on M.
+The Reconstruction Theorem combined with the
+Nerve Theorem tell that, for well-chosen values of r, η, the η-offsets of M are homotopy equivalent
```

Comment:
Please do a ctrl-f to replace this η character by $\eta$. Otherwise it doesn't compile.